jicofo-selector/src/main/resources/reference.conf

```diff
@@ -39,10 +39,18 @@ jicofo {
     //    Octo for geo-location).
     // IntraRegionBridgeSelectionStrategy: Use additional bridges when a bridge becomes overloaded (i.e. use Octo for
     //    load balancing).
+    // VisitorSelectionStrategy: Use separate strategies and bridges for visitors and participants.  In this case,
+    // visitor-selection-strategy and participant-selection-strategy should be defined
     //
     // Additionally, you can use the fully qualified class name for custom BridgeSelectionStrategy implementations.
     selection-strategy = SingleBridgeSelectionStrategy
 
+    // visitor-selection-strategy = SingleBridgeSelectionStrategy
```

Comment:
For whatever reason the default strategy in the file is SingleBridgeSelectionStrategy - I don't know why, probably because it's the right thing for a simple standalone instance?  So I thought I'd copy that.  But I guess you're right that it'd be better for the commented-out default to be the case that you'd want for the visitor case.

Reply:
I think we defaulted to SingleBridgeSelectionStrategy when we introduced octo and used it as a default just to be conservative. I think we should always use RegionBased and use the config flag to disable octo if that's what we want.